rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -509,6 +515,48 @@ void logPegoutConfirmed() {
         assertEvent(eventLogs, 0, BridgeEvents.PEGOUT_CONFIRMED.getEvent(), new Object[]{btcTx.getHash().getBytes()}, new Object[]{pegoutCreationRskBlockNumber});
     }
 
+    private static Stream<Arguments> logPegoutTransactionCreatedValidArgProvider() {
+        List<Arguments> args = new ArrayList<>();
+
+        args.add(Arguments.of(Sha256Hash.ZERO_HASH, Collections.singletonList(Coin.SATOSHI)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(5), coinListOf(500_000, 400_000, 1_000)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(10), Collections.singletonList(Coin.ZERO)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(10), Collections.EMPTY_LIST));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(15), null));
```

Comment:
```
btcTxHash = Sha256Hash.ZERO_HASH;
coinValue = Collections.singletonList(Coin.SATOSHI);
testArguments = Arguments.of(btcTxHash, coinValue);

args.add(testArguments);
```

and so on

Reply:
I think what you are proposing is redundant and will make the test cases super hard to follow. It will double the amount of lines that this method currently has. 

Btw, I also consider adding variables to something so obvious like `Collections.singletonList(Coin.SATOSHI);` it is way redundant and unnecessary.